PaolaSantolloV/react-apprenticeship-capstone1
# Mini challenge 3 and 5

## Description

This PR contain the Mini-Challenge 3: Hooks and Mini-Challenge 5: Routing
- Changes on the routing on the project.
- Add the useFetch custume hook.
- Add new test on the components and pages.
- Add login functionality
- Add favorites functionality


## Type of Change



## How should this be tested?
In your terminal run `npm install` after that `npm run start,` now you can see the page on [http://localhost:3000/]( http://localhost:3000/)
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## Any background context you want to provide?

If the request are over I could change the API-Key.

## What is the deliverable?
Third deliverable

## Screenshots and links

Netlify: [castoneproject-miniChallange3and5.com]( https://61e85e8a10a37100075baf57--silly-torvalds-2c8dd5.netlify.app/)
Coverage: 71.2%
![Screen Shot 2022-01-19 at 12 06 13](https://user-images.githubusercontent.com/97254325/150194722-3f609804-5751-4db1-8fdb-844dad8345ed.png)


## Checklist
